Cargo.toml

```diff
@@ -42,6 +42,7 @@ ethcore-light = { path = "ethcore/light" }
 ethcore-logger = { path = "logger" }
 ethcore-stratum = { path = "stratum" }
 ethkey = { path = "ethkey" }
+fetch = { path = "util/fetch" }
```

Comment:
Main crate is already able to instantiate `FetchClient` using a re-export from `hash-fetch` crate.